pages/get-involved.tsx

```diff
@@ -139,11 +139,6 @@ export default function GetInvolvedPage() {
         <HeroBanner
           header="Share your expertise with us"
           subheading="There are many ways to participate with the Clearviction team, and we appreciate all of them!"
-          overrideStyles={{
```

Comment:
I don't think it has to do with the removal of this specific code snippit, but the cards on the get involved page have different spacing on mobile and on desktop than both of their respective SOTs. Do you think you could adjust them while you are working on the overall HeroBanner component in this PR.